README.md

```diff
@@ -23,6 +23,33 @@ git clone https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git
 cd v2x-ros-conversion/docker
 ./build-image.sh
 ```
+
+To incorporate the v2x-ros-conversion package into a docker project, call the following command to clone the v2x-ros-conversion package into the project:
+```
+git clone --depth=1 https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git ${dir}/src/v2x-ros-conversion --branch "develop"
+```
+Once installed, the required dependencies for the v2x-ros-conversion package need to be installed prior to building the package. To do so, run 
+```
+sudo ~/src/v2x-ros-conversion/docker/install_dependencies.sh
+```
+
+Once the dependencies are installed, build the image as shown above.
+
+### Build From Source
+Ensure that a version of Ubuntu 22 is installed with the humble version of carma-base. Once installed, follow the following steps in install the v2x-ros-conversion package into the project.
```

Comment:
Same note as #17 . Need to check if build from source works without carma-base installed packages.